qiskit_experiments/framework/experiment_data.py

```diff
@@ -1485,7 +1487,11 @@ def _metadata_too_large(self):
         return sys.getsizeof(self.metadata) > 10000
 
     def save(
-        self, suppress_errors: bool = True, max_workers: int = 100, save_figures: bool = True
+        self,
+        suppress_errors: bool = True,
+        max_workers: int = 100,
```

Comment:
Can we also reduce this number? I think something like 3 workers is a more reasonable number. And the maximum allowed value should be capped so the user can't crash their system/the database.

Reply:
Done